language/stdlib/modules/Vector.move

```diff
@@ -1,6 +1,13 @@
 address 0x1 {
 
-/// A variable-sized container that can hold both unrestricted types and resources.
+/// A variable-sized container that can hold any type. Indexing is 0-based, and
+/// vectors are growable. This module has many native functions. We did not verify most of the
```

Comment:
Suggest to put the sentence about specification into a quote:

\> NOTE: We did not verify most of ... other modules as well.